src/workflows/airqo_etl_utils/tests/test_cloud_storage_streaming.py

```
@@ -0,0 +1,182 @@
+
+
+class FakeS3Body(io.BytesIO):
+    # behaves like boto3 StreamingBody for our tests
+    pass
+
+
+class FakeS3Client:
+    def __init__(self, csv_bytes: bytes):
+        self.csv_bytes = csv_bytes
+
+    def get_paginator(self, name):
```

Ruff (0.15.0)

[warning] Unused method argument: `name`

(ARG002)